src/app/services/baw-api/model.service.ts

```diff
@@ -0,0 +1,143 @@
+import { HttpClient } from "@angular/common/http";
+import { Inject, Injectable, InjectionToken } from "@angular/core";
+import { Subject } from "rxjs";
+import { AppConfigService } from "../app-config/app-config.service";
+import { APIErrorDetails } from "./api.interceptor";
+import { BawApiService, Filters } from "./base-api.service";
+
+export let MOCK_CLASS_BUILDER = new InjectionToken("test.class.builder");
```

Comment:
This seems like a conflation of concerns. Why is a test token defined in a non-test file? Is there a better spot for it?

Reply:
The injection token is used by the model.service.spec.ts file so moving this variable outside of the model.service file would be a bit awkward. Potentially i can move it to the injection token definition to the unit testing file and import it into the service file?

Reply:
I think my biggest issue is one of naming then. "Mocks" shouldn't exist in non-test code.